contrib/kube-prometheus/README.md

```diff
@@ -65,17 +67,17 @@ $ kubectl create -f manifests/
 
 The content of this project consists of a set of [jsonnet](http://jsonnet.org/) files making up a library to be consumed.
 
-Install this library in your own project with [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler#install):
+First create a new project, and install kube-prometheus as a library with [jsonnet-bundler](https://github.com/jsonnet-bundler/jsonnet-bundler#install) (the jsonnet package manager):
+
+> jsonnet-bundler or `jb` can be installed with `go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb`
```

Comment:
This seems redundant with the referenced `jsonnet-bundler#install` section and looks funky when rendered as a quote. I think we can leave `jb` as a prerequisite and point to the originating `jsonnet-bundler` repository.